README.md

```diff
@@ -302,7 +304,8 @@ view the [script options output][script_options] for the latest release.
           # Consider using a bot or group account for this token.
           phylum_token: ${{ secrets.PHYLUM_TOKEN }}
 
-          # NOTE: These are examples. Only one `github_token` entry line is expected.
+          # NOTE: These are examples. Only one `github_token` entry line is expected,
+          #       and only with comment generation enabled.
```

Comment:
Even when comment generation is enabled, the default can be used. So I think this wording is misleading.

```suggestion
          # NOTE: These are examples. At most one `github_token` entry line is needed.
```